script/GameObjects/Actions/Think.lua

```diff
@@ -1,7 +1,7 @@
 local Action = require('GameObjects.Action')
 local Player = require('GameObjects.Entities.Player')
 
-local kJobIterations = 5000 -- how many randomly-chosen jobs the asset will consider before deciding
+local kJobIterations = 4000 -- how many randomly-chosen jobs the asset will consider before deciding
```

Comment:
prob would be better to move this to a config or something

Reply:
Change made, update in follow-up commit.